// @mui/material-next/modern/index.js
/**
 * @mui/material-next v6.0.0-alpha.103
 *
 * @license MIT
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */
'use client';

export { default as Badge } from './Badge';
export * from './Badge';
export { default as Button } from './Button';
export * from './Button';
export { default as ButtonBase } from './ButtonBase';
export * from './ButtonBase';
export { default as Chip } from './Chip';
export * from './Chip';
export { default as FormControl } from './FormControl';
export * from './FormControl';
export { default as InputBase } from './InputBase';
export * from './InputBase';
export { default as Input } from './Input';
export { default as Slider } from './Slider';
export * from './Slider';
export { default as Tabs } from './Tabs';
export * from './Tabs';
export { default as Tab } from './Tab';
export * from './Tab';
export { default as TabScrollButton } from './TabScrollButton';
export * from './TabScrollButton';
export * from './styles';
